packages/lib/browser/browser.utils.ts

```diff
@@ -7,7 +7,7 @@ type BrowserInfo = {
 };
 
 export const getBrowserInfo = (): Partial<BrowserInfo> => {
-  if (!process.browser) {
+  if (typeof window === "undefined") {
```

Comment:
Perhaps it might make sense to add a small util `isBrowser()`? Makes that we have this logic only in one place, and also reads a bit easier. The change from NOT to now strict equals almost threw me off thinking it was reverted 😁